examples/post_ceph_hci/values.yaml

```diff
@@ -156,6 +156,7 @@ data:
         - name: tenant
           subnetName: subnet1
   deployment:
+    name: edpm-deployment-ipam-post-ceph
```

Comment:
im not entirely sure why ipam is in the name but otehrwas this change looks good to me